ResumableUpload.VB.Sample/My Project/Resources.resx

```diff
@@ -0,0 +1,117 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
You can safely delete this file, and several others in the sample projects.

- AssemblyInfo.vb
- AssemblyInfo.cs
- My Project\Resources.vb
- My Project\Resources.resx
- My Project\Settings

etc.